packages/react/src/hooks/useIsMountedState/useIsMountedState.ts

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from 'react'
 
-const useIsMounted = () => {
+export const useIsMountedState = () => {
```

Comment:
In my opinion, this hooks name is more readable for useIsMounted (original, or useCheckMounted)
Why you add sufix `State` to this hook's name?

Reply:
This comment is not valid now because of new commit b1d1a5c removing mounting check hook by usePrevious.

but I want to leave comment clarifing what my intention of previous code was

The reason for adding the subfix State to the name of the useIsMountedState hook is

1. useIsMountedOnThisRender: This hook only checks the mounted status of the component with ref and does not trigger rerender.
2. useIsMountedState: This hook changes the mounted state of the component and even rerenders it.

I wanted to clearly distinguish the roles of these two hooks by naming. An important distinction criterion is rerender.

In @suspensive/react we have to check the mounted state of the component in two ways.
1. Supsense.CSROnly should "rerender" and show children only when the component is mounted, but
2. ErrorBoundaryGroup only needs to check whether it is mounted or not, without rerender.

The reason I made this Pull Request was that the name useIsMounted was not enough to express the functionality of these two required hooks.

In two popular libraries, [@toss/react](https://slash.page/libraries/react/react/src/hooks/useIsMounted.i18n) and [usehooks-ts](https://usehooks-ts.com/react-hook/use-is-mounted), useIsMounted has two different functions as mentioned above.

As result, I thought there was no reason for a hook named useIsMounted to have a state and fire a rerender. Therefore, I would like to divide the names into useCheckMounted same with usehooks-ts's useIsMounted and useIsMountedState same with @toss/react's useIsMounted.